borg/testsuite/archiver.py

```diff
@@ -898,7 +898,7 @@ def test_list_list_format(self):
         self.cmd('create', test_archive, src_dir)
         output_1 = self.cmd('list', test_archive)
         output_2 = self.cmd('list', '--list-format', '{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}', test_archive)
-        output_3 = self.cmd('list', '--list-format', '{mtime:%s} {path}{NL}', test_archive)
+        output_3 = self.cmd('list', '--list-format', '{mtime:%s} {path}{NEWLINE}', test_archive)
```

Comment:
While I still don't know where NL came from in the old code I'm just going to revert this and add NL as an alias for NEWLINE. No one wants to type 10 characters for a new line anyway.